charts/dependency-track/templates/frontend/deployment.yaml

```diff
@@ -48,9 +48,8 @@ spec:
         env:
         - name: API_BASE_URL
           value: {{ .Values.frontend.apiBaseUrl | quote }}
-        {{- range $k, $v := .Values.frontend.extraEnv }}
-        - name: {{ $k }}
-          value: {{ $v | quote }}
+        {{- with .Values.apiServer.extraEnv }}
```

Comment:
Maybe `frontend` instead of apiServer` ?

Reply:
Good spot @niriak, a fix is on its way! #87